lwotgtelem/gnmit/subscribe/subscribe_test.go

```diff
@@ -0,0 +1,1415 @@
+//go:build !race
+
+/*
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
```

Comment:
I take it this entire thing is forked without modification from `openconfig/lemming/gnmi/subscribe/subscribe_test.go`, yes?

Reply:
Yes, it takes the code from there (that was originally github.com/openconfig/gribigo/gnmit I believe ;-))